terraform/prod1.tfvars

```diff
@@ -17,4 +17,4 @@ mesh_gateway      = "10.70.90.1"
 mesh_networkrange = "24"
 mesh_net_block    = "10.70.90.0"
 mesh_external_ip  = "199.170.132.45"
-meshdb_fqdn       = "db.mesh.nycmesh.net"
+meshdb_fqdn       = "db.mesh.nycmesh.net,forms.mesh.nycmesh.net,wiki.mesh.nycmesh.net"
```

Comment:
`wiki2` for now, and this can only happen after https://github.com/nycmeshnet/nycmesh-dns/pull/80 is merged

Reply:
```suggestion
meshdb_fqdn       = "db.mesh.nycmesh.net,forms.mesh.nycmesh.net,wiki2.mesh.nycmesh.net"
```